### PR TITLE
Travis support

### PR DESCRIPTION
**Pull request by frangarcj**
You need to define two environment variables in travis

$OAUTH - this is a personal access token from github with repo perms
$TAGPERM - this are github credentials in the form username:password

### CODE REVIEW

**Comment by d3m3vilurr** [comment]
probably oauth token can use $TAGPERM. id/pwd value can make problem if use two factor auth

**Comment by frangarcj** [comment]
right you can use oauth token for $TAGPERM

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+deploy:
+  provider: releases
+  api_key: $OAUTH 
+  file: 
+     - "build/taihen.skprx"
+     - "taihen.tar.gz"
+  skip_cleanup: true
+  on:
+    tags: false
+    all_branches: true
```
Comment (d3m3vilurr) [inline .travis.yml:40]: really need `all_branches` set?
how about change to `branch: master`

Comment (frangarcj) [reply .travis.yml:40]: Owner's taste

**Pull request merged**